run-dljc.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# This script runs the Checker Framework on each of a list of projects.
+
+### Usage
+
+# - Move this script to an experiment directory.
+# - Make a file containing a list of git repositories, one per line.
+# - Set variables in the "Configuration" section below.
+# - Run the following command:
+#     bash run-dljc.sh OUTDIR INLIST
+#
+# Where OUTDIR is a name for the experiment you're running, and
+# INLIST is the file containing list of git repositories.
+#
+# This script works as follows:
+# - Checkout each repository into OUTDIR/.
+# - Use do-like-javac to build it once, then
+#   replay the javac commands with checkers enabled.
+# - Place a copy of the results in the directory OUTDIR-results/.
+
+
+### Configuration
+
+# JAVA_HOME must point to a Java 8 JDK.
+if [ "x${JAVA_HOME}" = "x" ]; then
```

Comment:
It is better to write
```VARIABLE1="${VARIABLE1:-defaultvalue}```
because that permits users to just set environment variables to override defaults.

I suggest you recommend that and don't even mention the possibility of copying and editing this script.

It is probably even better just to have this script check that the variables are set rather than having default values, which is 1/3 of this script.